.vscode/settings.json

```diff
@@ -11,5 +11,8 @@
         "preorders",
         "undiscounted"
     ],
-    "python.linting.enabled": true
+    "python.linting.enabled": true,
+    "[python]": {
```

Comment:
I am not convinced that we should modify the default vscode settings. And if we really want to do that, we should do that in separate PR

Reply:
Changes reverted by amending [current commit](https://github.com/saleor/saleor/pull/13325/commits/3bc631a7eb5669670648eba18e187c82c03d81b6)